contributing/code/standards.rst

```diff
@@ -167,6 +167,24 @@ Service Naming Conventions
 
 * A group name uses the underscore notation.
 
+.. _contributing-code-standards-conditions:
+
+Conditions
+----------
+
+Use `Yoda condition`_ when checking a variable against a value to avoid an
+accidental assignment inside the condition statements::
```

Comment:
I think we should add this to the list of standards, instead of creating a custom section purely for one standard.

```rst
* Use `Yoda conditions`_ when checking a variable against a value to avoid an
  accidental assignment inside the condition statements.
```